tutorials/export/android_gradle_build.rst

```diff
@@ -57,3 +57,20 @@ configuration is needed.
 
     For example, ``_example/image.png`` will **not** be included as an asset,
     but ``_image.png`` will.
+
+Disable Swipe-to-Dismiss on Wear OS
+-----------------------------------
```

Comment:
```suggestion
Disabling Swipe-to-Dismiss on Wear OS
-------------------------------------
```
Slightly more idiomatic format for titles

Also, *possibly* also this. The current style guide for the manual is to avoid Actual Title Case for header titles, and only capitalize proper nouns and the first word. Seems like people refer to "swipe to dismiss" several ways, but here is [one instance](https://developer.android.com/training/wearables/views/exit#swipe-to-dismiss) of the official android docs using "swipe-to-dismiss"
```suggestion
Disabling swipe-to-dismiss on Wear OS
-------------------------------------
```